src/sonic-yang-models/yang-templates/sonic-acl.yang.j2

```diff
@@ -184,7 +184,7 @@ module sonic-acl {
 
 				leaf TCP_FLAGS {
 					type string {
-						pattern '0[x][0-9a-fA-F]{1,2}|0[X][0-9a-fA-F]{1,2}';
+						pattern '0[xX][0-9a-fA-F]{1,2}(/0[xX][0-9a-fA-F]{1,2})?';
```

Comment:
This pattern can match `0x24/0x24/0x24`, which is not expected.

Reply:
? will match 0 or 1 occurrence of the previous pattern.
0x24/ **0x24/0x24** will match 2 exceeding max 1 occurence.